src/util/paymentMethodHelper.js

```diff
@@ -72,3 +73,19 @@ export const validatePaymentMethods = (
 
 export const shouldShowGatewayComponent = (gateway, paymentMethods) =>
   !!paymentMethods.find(({ paymentGateway }) => paymentGateway === gateway);
+
+// returns common part between methods set in admin and those set
+// by Config.setVisibleAdyenPaymentMethods()
+export const getAvailablePaymentMethods = publisherPaymentMethods => {
+  const visibleAdyenPaymentMethods = JSON.parse(
+    getData('CLEENG_VISIBLE_ADYEN_PM') || '[]'
+  );
+
+  const availablePaymentMethods = visibleAdyenPaymentMethods.length
+    ? publisherPaymentMethods.filter(({ methodName }) =>
+        visibleAdyenPaymentMethods.includes(methodName)
+      )
+    : publisherPaymentMethods;
+
+  return availablePaymentMethods;
+};
```

Comment:
I think we should keep these payment methods (and logic in redux). Currently, we have state.publisherConfig.paymentMethods which represents payment methods set in cleeng admin. I think that when publisher sets visible payment methods we could save them in the state.publisherConfig.visiblePaymentMethods. It's an improvement, not a must-have, but worth to validate :)